### PR TITLE
Revised readme file and added two logo pictures

### PR DESCRIPTION
**Pull request by yiiihao**
Added pic folder with two logo pictures and revised readme file as required on coursework. Please check the content. Unable to make the two logos side by side (works fine locally but cannot display properly on github).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,61 +1,23 @@
...
-
-### Fetch data
-```
-python -m capstone fetch
-```
-
-### Run notebooks
-```
-jupyter notebook notebooks/
-```
```
Comment (vishu-tyagi) [inline README.md]: I think we should keep the setup instructions as they are, just move them below the project description.

File: README.md [hunk 2]
```diff
@@ -1,61 +1,23 @@
...
+<img src="pic/Johnson-Johnson-Logo.png" alt="image" title="Text to show on mouseover" width='270'/> <img src="pic/columbia_dsi_logo.png" alt="image" title="Text to show on mouseover" width="270"  />
+
+#### Problem Statement/ Business Need: 
+
+The GxP regulatory environment is very complex as different countries have their own regulations, and standardization is very limited. GxP regulations and guidance documents are thousands of pages of text files (pdf or HTML) posted in several internet locations. These regulatory requirements have to be manually parsed, analyzed, and classified to develop the J&J quality requirements. This is a time-consuming process.
+
+#### Project Outcome / Solution:
+
+- With fine-tuned GPT-3 model, classify requirements by quality topics and classify quality topics requirements into themes; summarize theme requirements into a J&J Quality requirement that meets all the regulations and guidance documents.
+- Build metrics to evaluate the model and benmark using other available large language models as well as traditional machine learning models.
```
Comment (vishu-tyagi) [inline README.md]: Typo here: benmark should be benchmark

File: README.md [hunk 3]
```diff
@@ -1,27 +1,32 @@
 # JNJ Capstone Project
+
+
+
 <p float="left">
-  <img src="pic/Johnson-Johnson-Logo.png" width="270" />
-  <img src="pic/columbia_dsi_logo.png" width="270" /> 
+  <img src="pic/JNJ_logo.png" width='300' />
+  <img src="pic/columbia_dsi_logo.png" width='300'/>
```
Comment (vishu-tyagi) [inline README.md]: Looks great

File: README.md [hunk 4]
```diff
@@ -1,61 +1,23 @@
...
-```
-python -m capstone fetch
-```
-
-### Run notebooks
-```
-jupyter notebook notebooks/
-```
+# JNJ Capstone Project
+<img src="pic/Johnson-Johnson-Logo.png" alt="image" title="Text to show on mouseover" width='270'/> <img src="pic/columbia_dsi_logo.png" alt="image" title="Text to show on mouseover" width="270"  />
```
Comment (vishu-tyagi) [inline README.md]: Nice idea to add logos

File: README.md (hunk 2 above)
Comment (vishu-tyagi) [inline README.md]: Content looks great

**Review by vishu-tyagi — CHANGES_REQUESTED** [review]
Hi Yihao. I've left two comments that require your attention. One regarding setup instructions on the readme file and other about a slight typo. 

Also, I think it might be a good idea to keep images in /notebooks/images/ rather than at the top level directory. Could you please move the images from /pic/ to /notebooks/images/?

File: README.md (hunk 2 above)
Comment (yiiihao) [reply README.md]: Fixed typo

File: README.md (hunk 1 above)
Comment (yiiihao) [reply README.md]: Yes. Added setup instructions.

**Comment by yiiihao** [comment]
Added setup instructions below the project description, fixed typo, moved images to notebooks/images/.

**Review by vishu-tyagi — APPROVED** [review]

**Review by siwenxie919 — APPROVED** [review]
Good job!

**Pull request merged**